src/main/java/helpers/DriverHelper.java

```diff
@@ -5,13 +5,16 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
 
+import java.util.concurrent.TimeUnit;
+
 public class DriverHelper {
 
     private static WebDriver driver;
 
     public static void initDriver() {
         WebDriverManager.chromedriver().setup();
         driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Constants.TIME_WAIT, TimeUnit.SECONDS);
```

Comment:
We should not use implicit wait in project, please research more about this wait

Reply:
I've already fixed it.